relentless/simulate/simulate.py

```diff
@@ -109,3 +112,184 @@ class Data:
     @abc.abstractmethod
     def __call__(self, sim):
         pass
+
+class Potentials:
+    """Combination of multiple potentials.
+    """
+    def __init__(self):
+        self._pair = PairPotentialTabulator(None,None)
+
+    @property
+    def pair(self):
+        return self._pair
+
+class PotentialTabulator:
```

Comment:
This needs docs. Testing is a little tricky because we don't have potentials yet that work with this.